waspc/data/Generator/templates/server/src/routes/apis/index.ts

```diff
@@ -9,32 +9,32 @@ import { type AuthUser } from 'wasp/auth'
 {=/ isAuthEnabled =}
 
 {=# apiNamespaces =}
-{=& namespaceMiddlewareConfigFnImportStatement =}
+{=& namespaceMiddlewareConfigFn.importStatement =}
 {=/ apiNamespaces =}
 
 {=# apiRoutes =}
-{=& importStatement =}
+{=& apiRouteFn.importStatement =}
 {=# routeMiddlewareConfigFn.isDefined =}
 {=& routeMiddlewareConfigFn.importStatement =}
 {=/ routeMiddlewareConfigFn.isDefined =}
 {=/ apiRoutes =}
 
 const idFn: MiddlewareConfigFn = x => x
 
-{=# apiRoutes =}
-{=^ routeMiddlewareConfigFn.isDefined =}
-const {=& routeMiddlewareConfigFn.importAlias =} = idFn
```

Comment:
No need for custom aliases anymore, rewrote this code to work with the new mangling.